menu.py

```diff
@@ -18,3 +20,57 @@ def __init__(self):
 
     def quit(self):
         self.destroy()
+
+    def easy_click(self):
+        PlayField.ROWS = int(6)
+        PlayField.COLUMNS = int(6)
+        PlayField.MINES = int(10)
```

Comment:
Немає потреби обгортати число в `int()`
```suggestion
        PlayField.MINES = 10
```
Тут і в інших місцях.